includes/object-cache.php

```diff
@@ -284,6 +284,13 @@ class WP_Object_Cache {
      */
     public $cache = [];
 
+    /**
+     * Holds cache key to redis key map
+     *
+     * @var array
+     */
+    public $keys = [];
```

Comment:
Since it's centrally managed, should we use a `static $keys = [];` inside the method itself?